plugin.video.worldstarhiphop/addon.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+#
+# Imports
+#
+# the YDStreamExtractor needs to be before the future imports. Otherwise u get an 'check hostname' error.
+import YDStreamExtractor
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+import sys
+import urllib.parse
+import xbmc
+
+from resources.lib.worldstarhiphop_const import ADDON, DATE, VERSION, BASEURL
+
+# Parse parameters...
+if len(sys.argv[2]) == 0:
+    #
+    # Main menu
+    #
+    xbmc.log("[ADDON] %s, Python Version %s" % (ADDON, str(sys.version)), xbmc.LOGDEBUG)
```

Comment:
```suggestion
    xbmc.log(f"[ADDON] {ADDON}, Python Version {sys.version}", xbmc.LOGDEBUG)
```

You might want to have a look into python3 formatted strings in the future

Reply:
Done